shared/middleware/files_api.rb

```diff
@@ -47,7 +47,7 @@ def can_view_abusive_assets?(encrypted_channel_id)
   def codeprojects_can_view?(encrypted_channel_id)
     owner_storage_id, _ = storage_decrypt_channel_id(encrypted_channel_id)
     owner_user_id = user_storage_ids_table.where(id: owner_storage_id).first[:user_id]
-    get_user_sharing_disabled(owner_user_id)
+    !get_user_sharing_disabled(owner_user_id)
```

Comment:
This was the real bug that caused web lab sharing to break

Reply:
I've made mistakes like this many times before. One thing I try to do to avoid it is to stick to positive names for functions/variables when possible (i.e. `get_user_sharing_enabled`). No need to change this here, but something to consider going forward.